src/plugins/intel_cpu/src/compiled_model.cpp

```diff
@@ -103,16 +117,36 @@ CompiledModel::CompiledModel(const std::shared_ptr<ov::Model>& model,
     } else {
         CompiledModel::get_graph();
     }
-    // init sub stream threads of executor
-    int sub_streams = m_cfg.streamExecutorConfig.get_sub_streams();
-    if (sub_streams > 0 && stream_executor != nullptr) {
-        std::vector<Task> tasks;
-        tasks.resize(sub_streams);
-        for (auto&& task : tasks) {
-            task = [] {};
+    std::cout << "xxxxxxxxx: m_subCompileModel: " << m_cfg.enableSubStreams << ", " << m_cfg.streamExecutorConfig.get_sub_streams() << "\n";
+    if (m_cfg.enableSubStreams) {
+        m_cfg.enableSubStreams = false;
```

Comment:
Why not clone m_cfg to avoid potential race condition?

Reply:
Ok, I cloned m_cfg to create sub compiled model.